ui/lesson/Terminal.tsx

```diff
@@ -21,27 +23,23 @@ export default function Terminal({ success, lines, next, onChange }) {
     >
       <div className="flex grow flex-col items-stretch text-white">
         <div className="flex grow">
-          <ReactTerminal
-            name="Saving Satoshi"
-            colorMode={ColorMode.Light}
-            onInput={onChange}
-          >
+          <TerminalInterface onInput={onChange}>
             {lines.map((line, index) => (
               <span
                 key={index}
-                className={`${
-                  (line.type === 'answer' &&
-                    'react-terminal-answer text-green') ||
-                  (line.type === 'output' &&
-                    'react-terminal-output text-[var(--terminal-output)]') ||
-                  (line.type === 'input' &&
-                    'react-terminal-previous-input text-white/80')
-                }`}
+                className={clsx({
+                  'react-terminal-answer text-green': line.type === 'answer',
+                  'react-terminal-output text-[var(--terminal-output)]':
+                    line.type === 'output',
+                  'react-terminal-previous-input text-white/80':
+                    line.type === 'input',
+                  'mt-2.5': index != 0,
+                })}
               >
                 <TerminalOutput>{`${line.value}`}</TerminalOutput>
```

Comment:
I noticed `TerminalOutput` component is a single line of jsx, think it makes sense to just put that code here instead of having a dedicated component for it.